dcrjson/chainsvrcmds.go

```diff
@@ -107,14 +107,43 @@ type EstimateFeeCmd struct {
 	NumBlocks int64
 }
 
-// NewEstimateFeeCmd returns a new instance which can be used to issue a
+// NewEstimateFeeCmd returns a new instance which can be used to issue an
 // estimatefee JSON-RPC command.
 func NewEstimateFeeCmd(numBlocks int64) *EstimateFeeCmd {
 	return &EstimateFeeCmd{
 		NumBlocks: numBlocks,
 	}
 }
 
+// EstimateSmartFeeMode defines estimation mode to be used with
+// the estimatesmartfee command.
+type EstimateSmartFeeMode int
+
+const (
+	// EstimateSmartFeeEconomical returns an
+	// economical result.
+	EstimateSmartFeeEconomical EstimateSmartFeeMode = iota
```

Comment:
As mentioned this would be better as a string, so with that change, this comment will become moot, but values used in RPC APIs generally shouldn't use `iota` because they need to be stable, unlike internal APIs where it doesn't really matter if the actual numbers change.